wallet/src/de/schildbach/wallet/ui/WalletUri.java

```diff
@@ -153,9 +153,7 @@ public BitcoinURI toBitcoinUri() throws BitcoinURIParseException {
         Coin amount = getAmount();
         String sender = getSender();
         String bitcoinUri = BitcoinURI.convertToBitcoinURI(address, amount, sender, null);
-        if (useInstantSend() || forceInstantSend()) {
-            bitcoinUri += "&" + BitcoinURI.FIELD_INSTANTSEND + "=1";
-        }
+
```

Comment:
The InstantSend field is obsolete.  This was from a time when instantSend had to be requested and there was an extra fee for using it.